api/src/main/java/org/openmrs/api/db/hibernate/HibernateFormDAO.java

```diff
@@ -463,10 +463,10 @@ private Criteria getFormCriteria(String partialName, Boolean published, Collecti
 			DetachedCriteria subquery = DetachedCriteria.forClass(FormField.class, "ff");
 			subquery.setProjection(Projections.property("ff.form.formId"));
 			subquery.add(Restrictions.in("ff.formFieldId", anyFormFieldIds));
-			crit.add(Subqueries.propertyIn("form.formId", subquery));
```

Comment:
I changed this back from "f.formId" to "form.formId" but your unit test did not fail.

Reply:
Haaaa, nope. I fails. Let me just make it fail here just deliberatly

Reply:
That build failure is a prof to show that "I changed this back from "f.formId" to "form.formId" but your unit test did not fail." Could be you missed something.

Reply:
This is what am saying. Take a look at line 466 above. When i change f.formId back to form.formId as it was, none of your tests fails.

Reply:
Are you sure no test fails. This time round am sure at least. A test fails @dkayiwa . Try it locally